client/my-sites/backup/backup-date-picker/test/index.jsx

```diff
@@ -56,13 +61,14 @@ describe( 'BackupDatePicker', () => {
 
 	test( "Shows 'Yesterday' as the previous date if the current date is today", () => {
 		const today = moment();
+		render( <BackupDatePicker selectedDate={ today } onDateChange={ () => {} } /> );
 
-		const picker = shallow( <BackupDatePicker selectedDate={ today } onDateChange={ () => {} } /> );
-
-		const previousDate = picker.find(
-			'.backup-date-picker__select-date--previous .backup-date-picker__display-date'
+		const yesterdayTextNode = screen.getByText( 'Yesterday' );
```

Comment:
Just a nitpick but is there a need for a variable `yesterdayTextNode` here? I like to read it being explicit and idiomatic like `expect( screen.queryByText( 'WordPress.com' ) ).toBeInTheDocument();`

Reply:
Storing it in a variable because we do multiple assertions in that test, else we will have to do redundant query calls